docusaurus/docs/Android/02-client/06-guides/01-push-notifications/01-overview.mdx

```diff
@@ -35,14 +35,30 @@ ChatClient.Builder("api-key", context)
     .build()
 ```
 
+:::note
+You can disable push notifications client-side by providing custom `NotificationConfig` with `pushNotificationsEnabled` set to `false`.
```

Comment:
Aren't they disabled by default then, as per the changelog? Then the docs should say "disabled by default, *here's* how you enable them".

Reply:
let me think about better wording here